packages/browser/src/node/index.ts

```diff
@@ -23,18 +29,81 @@ export default (project: WorkspaceProject, base = '/'): Plugin[] => {
         }
       },
       async configureServer(server) {
+        const [testerHtml, runnerHtml, injectorJs] = await Promise.all([
```

Comment:
I guess you can defer resolving promises when required, you don't need to block vitest startup/configuration here, middlewares are async, declare them after distRoot declaration (I guess you can also split `Promise.all` into individual promises)